lib/battle/IBattleInfoCallback.h

```diff
@@ -43,6 +43,12 @@ struct DamageEstimation
 	DamageRange kills;
 };
 
+struct SpellEffectValue
+{
+	int64_t hpDelta = 0;    // +heal / -damage
+	int32_t unitsDelta = 0; // +resurrected / -killed
+};
+
```

Comment:
To be honest this does not sounds to be different from existing `DamageEstimation`, but rather one that represents fixed damage.

Sure, this is not necessarily *damage*, but also includes heal. But other than that? Sounds like the same entity. So perhaps reuse existing `DamageEstimation` struct?